src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -119,11 +122,11 @@
   public Label mostRecentAchievementNameLabel;
   public Pane lockedAchievementsContainer;
   public Pane unlockedAchievementsContainer;
-  public ToggleButton globalButton;
-  public ToggleButton ladder1v1Button;
   public NumberAxis yAxis;
   public NumberAxis xAxis;
-  public LineChart<Integer, Integer> ratingHistoryChart;
+  public LineChart<Long, Integer> ratingHistoryChart;
+  public ComboBox<String> timePeriodComboBox;
+  public ComboBox<String> ratingTypeComboBox;
```

Comment:
```suggestion
  public ComboBox<RatingTypeEnum> ratingTypeComboBox;
```

Reply:
Hmm yeah I used the Strings so that it can use i18n I am not sure if an enum can use that

Reply:
Well u register a ValueFactory in which u do ()->i18n.get(item.getI18nKey()) after u define such a property on the enum

Reply:
Something along that lines